modules/virt-about-virtualization-overview.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assembly:
+//
+// * virt/logging_events_monitoring/virt-reviewing-vm-dashboard.adoc
+//
+
+[id="virt-about-reviewing-top-consumers_{context}"]
+= About reviewing top consumers
+
+In the {VirtProductName} dashboard, you can select a specific time period and view the top consumers of resources within that time period. Top consumers are virtual machines or `virt-launcher` pods that are consuming the highest amount of resources.
```

Comment:
This is about the CNV Dashboard and not the Virtualization Overview, correct?

Reply:
This module has been removed - please ignore.